feat: add page descriptions and update titleType (#340)
carbon-design-system/carbon-website: src/pages/components/form/code.mdx

---
title: Form

> description: Forms are used for submitting data so be as concise as possible when designing. Think about each field and what value the data will provide.

Reviewer: ```suggestion
description: Forms are used for submitting data. Consider each field and what value the data will provide.
```
